src/materialized/src/bin/materialized/main.rs

```diff
@@ -378,6 +379,10 @@ pub struct Args {
         hide = true
     )]
     frontegg_api_token_url: Option<String>,
+    /// Enable cross-origin resource sharing (CORS) for HTTP requests from the
+    /// specified origin.
+    #[structopt(long, env = "MZ_CORS_ALLOWED_ORIGIN", hide = true)]
+    cors_allowed_origin: Vec<HeaderValue>,
```

Comment:
interesting, how do you provide multiple values on the environment? JSON-encode? comma-separate?

Reply:
Specify the same arg multiple times! This is what we do in other places (`—orchestrator-service-label`, for example).

Reply:
Ugh, yeah, that's a good point. Guess so. Can fix in post, for sure, if there's a better way!

Reply:
we'll probably not get into a situation where we have to set multiple; and if we do, yep: post-production it is (: